### ISSUE TITLE
Make the UI less compact on desktop browsers

### ISSUE THREAD

**Issue by sbraz** [airdcpp-web/airdcpp-webclient#117]
I understand that the reason why there are so many submenus is because the UI is also optimised for small devices but I think desktop users could benefit from a few changes (that would only be enabled for desktop user-agents) :
* don't put settings/favorites/logout in a submenu
* don't put the filelist options in a submenu, make it more like the chat options:
  * add an "open new" button in the filelists tab, similar to the conversation one
  * display all open filelists on the left side of the window, again this would be similar to the chat windows
* same thing for the "view file" windows

**Comment by maksis** [comment]
> don't put settings/favorites/logout in a submenu

Displaying that many large text-only menu buttons on the top would look terrible in my opinion. Not being able to fit the main on screen would also force switching to mobile layout, which isn't something that should be done too soon. The UI should also be kept in a state that new menu buttons can be added easily without worrying too much about the width. Dropdown items in navigation menus aren't really that uncommon on web sites targeted for desktop devices either (just look at Github...).

What do you usually need to access from Favorite or Settings? I've been considering moving of shared directories from settings to a separate main view as it's not that different from favorite hubs and it would make refreshing the share easier (and more information could be displayed as well).

> don't put the filelist options in a submenu, make it more like the chat options
> same thing for the "view file" windows

Those session layouts are different from chat views mainly because there's less need to browse though sessions, no notifications from other sessions and they benefit more from horizontal space (I know, there are users who disagree about this what comes to chat views). 

I'm not totally happy with the usage of those layouts but just displaying the number of open sessions would probably make them feel more natural....

**Comment by sbraz** [comment]
What kind of screen resolution are we talking about? I have 1600 pixels in width and 80% of the menu bar is empty. Those are just two tiny icons that would take very few pixels.
Yes it's mostly the shares that I access there. Moving them to another view would be nice too.

How is horizontal space an issue for file lists? You could use a bit more horizontal space since the left part of the window isn't really useful (and that's ~ 800 pixels for me)

**Comment by maksis** [comment]
> What kind of screen resolution are we talking about? I have 1600 pixels in width and 80% of the menu bar is empty. Those are just two tiny icons that would take very few pixels.
Yes it's mostly the shares that I access there. Moving them to another view would be nice too.

All icons used to be on the top earlier and it looked/felt terrible (even on a 4k monitor), which was another important reason for me to change it. If you look at other sites, such as Github, Google or Facebook, the navigation bar isn't filled with huge number of menu items. The dropdown also creates a separation between configuration items and other menu items.

> How is horizontal space an issue for file lists? You could use a bit more horizontal space since the left part of the window isn't really useful (and that's ~ 800 pixels for me)

The total width of the sidebar (including the menu) is already about the same as the width of the main layout of many desktop sites (such as Github). 

I'd just prefer thinking of other alternatives before making everything wider (and I think that there's something that can be done). Keeping the layouts somewhat unified on different screen size also makes my work easier. since primarily using a 4k monitor for development already makes it easier to disregard smaller devices.

You can test how the layout behaves on different screens by resizing your browser window.

**Comment by gangefors** [comment]
Just wanted to add my two cents to this old discussion since I also see a need for faster access to some items in the dropdown.

I find myself visiting Share and Settings quite frequently, way more frequently than Transfers for example, and I would love to have them directly accessible. I agree that there are some items that might not be needed in the main header and can be put into the dropdown menu. But what items those are can be very different depending on each person's usage.

Maybe it would be possible to mark a menu item as "pinned" and have "un-pinned" items in the dropdown menu?
